### ISSUE TITLE
Test failures on Win32

### ISSUE THREAD

**Issue by pdl** [pdl/Articulate#35]
Tests on Win32 failed:
- http://www.cpantesters.org/cpan/report/c4bc10a0-6bfe-1014-8e9d-369f0b0bf8b3
- http://www.cpantesters.org/cpan/report/382bfb31-6bfe-1014-ad60-2b9967be6cac

Failed with:
```
Error:  Invalid default 'An unknown error has occurred' for Articulate::Error->simple_message - not a coderef or code-convertible object at C:/strawberry5162/perl/site/lib/Method/Generate/Accessor.pm line 57.
```

**Comment by pdl** [comment]
Looks like the immediate caused could be resolved by requiring a reasonably new version of Moo. There are probably some genuine Win32 bugs behind it though, we just can't see.

**Comment by pdl** [comment]
Seems to be fixed in v0.004:

- http://matrix.cpantesters.org/?dist=Articulate%200.004;os=mswin32;perl=5.16.2;reports=1
